test/yamlSchemaService.test.ts

```diff
@@ -67,6 +67,50 @@ describe('YAML Schema Service', () => {
       expect(schema.schema.type).eqls('array');
     });
 
+    it('should handle schemas that use draft-04', async () => {
+      const content = `openapi: "3.0.0"
```

Comment:
I specifically wanted to test an openapi v3.0.0 schema because that's what is not working in my IDE.